### DISCUSSION TITLE
Ability to choose a label, release version or some other structured data type from an issue form dropdown

### DISCUSSION THREAD

**Discussion by labrose** [community/community#4266] [Projects and Issues]
When creating an issue form dropdown, I'd like the submitter to be able to choose from a set of existing labels, release versions or some other structured data type. This choice would then be reflected in their submission and the correct metadata added to the issue.

**Comment by jeherve** [comment]
I think that would be a nice addition indeed. If a dropdown could match a label, it would allow issue forms to be used to triage issues as they are created in a repository.

Until this is implemented, a potential workaround can be to use GitHub actions to automatically add a label to an issue based on the issue's body (and thus the heading and dropdown choice). Here is an example:
https://github.com/Automattic/jetpack/blob/1be27985f8c9fcea91dacdaf5bffbd300bc1c3c0/projects/github-actions/repo-gardening/src/tasks/triage-new-issues/index.js

**Comment by Kixunil** [comment]
This would be a reasonable alternative to allowing public to set labels.

**Comment by Bibo-Joshi** [comment]
Sounds related to #5642

**Reply by dralley** [reply]
And https://github.com/github/roadmap/issues/277

**Comment by chipzoller** [comment]
I would want to explicitly define the mapping in a given drop-down for the displayed value and corresponding label because they may not be the same. Ex., display in a drop-down "Windows" and, if this value is chosen, automatically apply label "win_x86" to the issue.
